pkg/utils/openshift_context.go

```diff
@@ -20,6 +26,36 @@ type OpenshiftContext struct {
 	OpenshiftFlavor OpenshiftFlavor
 }
 
+func NewOpenshiftContext(config *rest.Config, scheme *runtime.Scheme) (*OpenshiftContext, error) {
+	if ClusterType != ClusterTypeOpenshift {
+		return nil, nil
+	}
+
+	mcclient, err := mcclientset.NewForConfig(config)
+	if err != nil {
+		return nil, err
+	}
+
+	openshiftFlavor := OpenshiftFlavorDefault
+	if ClusterType == ClusterTypeOpenshift {
```

Comment:
Maybe I miss something but if the first conditional in this function checks for `ClusterType != ClusterTypeOpenshift`, how can this ever be false?

Reply:
you are right I change it

Reply:
I don't see the change